src/main/resources/application.conf

```diff
@@ -170,7 +170,7 @@ mantis {
 
       # Enabled JSON-RPC APIs over the JSON-RPC endpoint
       # Available choices are: eth, web3, net, personal, daedalus
-      apis = "eth,web3,net,daedalus"
+      apis = "eth,web3,net,personal,daedalus"
```

Comment:
so, personal and daedalus will be enabled by default?

What about handling this in https://github.com/input-output-hk/mantis/pull/331 ?

Reply:
This was discussed with @mcsherrylabs